src/core/public/chrome/chrome_service.tsx

```diff
@@ -165,12 +167,41 @@ export class ChromeService {
     // Can delete
     const getNavType$ = uiSettings.get$('pageNavigation').pipe(takeUntil(this.stop$));
 
+    const isIE = () => {
+      const ua = window.navigator.userAgent;
+      const msie = ua.indexOf('MSIE '); // IE 10 or older
+      const trident = ua.indexOf('Trident/'); // IE 11
+
+      return msie > 0 || trident > 0;
+    };
```

Comment:
It might be safer to use the battle-tested [browserslist-useragent](https://github.com/browserslist/browserslist-useragent) package:

```ts
const { matchesUA } = require('browserslist-useragent')

const isIE = matchesUA(
  window.navigator.userAgent,
  { browsers: ['ie', 'ie_mob'] }
);
```

Reply:
Excellent. Didn't know that was available. I'll switch it up.

Reply:
Josh and I did some back and forth over Slack. This lib isn't usable by the browser. We're going to continue with what I had originally since it seems like the best option. I've done testing to make sure it works as you'd expect.